velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -93,35 +102,11 @@ class SubstraitVeloxPlanConverter {
     return lengths_;
   }
 
- private:
-  /// The Partition index.
-  u_int32_t partitionIndex_;
-
-  /// The file paths to be scanned.
-  std::vector<std::string> paths_;
-
-  /// The file starts in the scan.
-  std::vector<u_int64_t> starts_;
-
-  /// The lengths to be scanned.
-  std::vector<u_int64_t> lengths_;
-
-  /// The unique identification for each PlanNode.
-  int planNodeId_ = 0;
-
-  /// The map storing the relations between the function id and the function
-  /// name. Will be constructed based on the Substrait representation.
-  std::unordered_map<uint64_t, std::string> functionMap_;
-
-  /// The Substrait parser used to convert Substrait representations into
-  /// recognizable representations.
-  std::shared_ptr<SubstraitParser> subParser_{
-      std::make_shared<SubstraitParser>()};
-
-  /// The Expression converter used to convert Substrait representations into
-  /// Velox expressions.
-  std::shared_ptr<SubstraitVeloxExprConverter> exprConverter_;
+  /// Looks up a function by ID and returns function name if found. Throws if
```

Comment:
Does this return function name or a string containing function name and input types: <name>_<arg_type1>_<arg_type2>...?

Reply:
Yes, it does. Added more notes there.